.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/*
```

Comment:
- [ ] Kindly change the content of the file. Your .gitignore file should contain node_modules AND test.md file. 
The content of the git ignore should be: 
```
node_modules
test.md
```
🤓

Reply:
alright on it